RELEASE.md

```diff
@@ -0,0 +1,13 @@
+## Release process:
+
+1. Create new branch
+
+- update s.version to x.y.z in [FishjamCloudClient.podspec](./FishjamCloudClient.podspec)
+- update s.version to x.y.z in [RNFishjamClient.podspec](./packages/react-native-client/ios/RNFishjamClient.podspec)
+- update version to x.y.z in [package.json](./packages/react-native-client/package.json)
+- update version to x.y.z in [package.json](./package.json)
+- update version to x.y.z in [build.gradle](./packages/react-native-client/android/build.gradle)
```

Comment:
I want a script to do this automatically, no way I'm editing manually 5 files

Reply:
For now i would leave it as it is. If this way will work as expected, then script can be created. Another thing is that there will be more files to update ( e.g. `read.me`, documentation in future) in almost every package. It would require some fixed/known structure.